tests/install_upgrade_operators/crds_cluster_readers_role/test_crds_cluster_readers_role.py

```
@@ -50,18 +55,23 @@ def get_cnv_crds(admin_client: DynamicClient) -> list[CustomResourceDefinition]:
     ]
 
 
-@pytest.fixture(scope="module")
-def jira_76680_open():
-    return is_jira_open(jira_id="CNV-76680")
+@pytest.fixture()
```

Ruff (0.14.13)

Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)